model/vm_host.rb

```diff
@@ -350,6 +350,19 @@ def check_storage_kernel_logs(ssh_session, devices)
     error_count.empty?
   end
 
+  def check_clock_source(ssh_session)
+    clock_source = ssh_session.exec!("cat /sys/devices/system/clocksource/clocksource0/available_clocksource")
+    return false unless clock_source.exitstatus == 0
```

Comment:
Is this an expected condition? If not, shouldn't we treat it similarly to unexpected clock source (including the emit)?